src/app/services/persistence.service.js

```diff
@@ -0,0 +1,19 @@
+angular.module('librarium')
+  .factory('UserPersistence', ['$cookies', function ($cookies) {
+    let userSession = { loggedin: false };
```

Comment:
there's no need to create an extra attribute, the user is logged if userSession is an object, otherwise it should be null or undefined, so initialize the userSession variable as such